HumanControls/JoystickController.h

```diff
@@ -0,0 +1,31 @@
+#ifndef JOYSTICKCONTROLLER_H
+#define JOYSTICKCONTROLLER_H
+
+#include "Arduino.h"
+
+class JoystickController
+{
+public:
+    //deadZoneSize is the total proportion of the joystick's range of values to omit
+    JoystickController(unsigned int xPin,
+                       unsigned int yPin,
+                       unsigned int xDeadZoneSize,
+                       unsigned int yDeadZoneSize,
+                       unsigned int xMin,
+                       unsigned int xMax,
+                       unsigned int yMin,
+                       unsigned int yMax);
+    void update();
+    double getX();
+    double getY();
+
+private:
+    double calcX(int rawValue);
+    double calcY(int rawValue);
+
+    int xPin, yPin, xMin, xMax, yMin, yMax;
```

Comment:
Let's try to consolidate on `m_` prefixes for instance variable names.

Reply:
Done.